index.html

```diff
@@ -64,5 +64,183 @@ <h2 class="container job-title">I am a Software developer </h2>
             </ul>
         </div>  
     </section>
+    <!-- Recent work section -->
+
+<section class="recent-works">
+    <div class="first">
+        <h3>My Recent Works</h3>
+        <hr>
+    </div>
+    <div class="recent-work" id="recent-works">
+        <a href="" class="women"><img src="assets/Img_Placeholder.png" alt="women doing yoga"></a></div>
+    <div>
```

Comment:
- Please check the path to this image as it is not displayed properly on the webpage.
- Please check the font family used on the Figma design and please use it appropriately. 

##### Figma design 
![Screenshot from 2022-06-04 10-17-32](https://user-images.githubusercontent.com/58771507/171993028-ea56e6ac-8da7-4cec-a406-2dcf4d4f2eb5.png)

##### Your design
![Screenshot from 2022-06-04 10-17-05](https://user-images.githubusercontent.com/58771507/171993041-17e592f9-0edd-48a3-844d-d1b9c4314137.png)